### ISSUE TITLE
[5.1] [a11y] Treeselect menu arrow icons

### ISSUE THREAD

**Issue by C-Lodder** [joomla/joomla-cms#43068]
### Steps to reproduce the issue

In dark mode, the treeselect arrow icons have two issues:

1. The colour doesn't meet the necessary contrast requirements in dark mode
2. They're interactive elements, however are not focusable

### Screenshot

![image](https://github.com/joomla/joomla-cms/assets/2019801/a5e115b4-9187-4004-b221-052f5a052567)

**Comment by brianteeman** [comment]
confirmed

**Comment by brianteeman** [comment]
Should now be resolved with #43125 and #43160

**Comment by alikon** [comment]
please test with https://github.com/joomla/joomla-cms/pull/43125 and https://github.com/joomla/joomla-cms/pull/43160
